### DISCUSSION TITLE
有没有让ollama上的本地模型支持调用插件（函数调用）的方法？

### DISCUSSION THREAD

**Discussion by muerdon** [mushan0x0/AI0x0.com#558] [Q&A]
有哪些开源模型支持调用插件？

**Comment by mushan0x0** [comment]
https://ollama.com/search?c=tools

**Reply by muerdon** [reply]
我问的是在AI0x0对话窗口要怎么调用插件，至少我在qwen2.5，glm4，gemma2这几个模型上没法调用插件，插件栏里的选项都是非激活状态。。 只有选用在线api模式时，插件栏里的框框才能打上钩。。。
